package-lock.json

```diff
@@ -1,9274 +1,15 @@
 {
   "name": "findologic",
   "version": "1.0.0",
-  "lockfileVersion": 2,
```

Comment:
Oof you made a mistake. Please revert this file to how it was before and make sure that your npm and node versions are up to date. Here are mine (they are not on the latest versions, but will produce a version 2 lock file):

![image](https://user-images.githubusercontent.com/20555112/155354852-f3a45790-0af6-46ed-adc6-88a7d041f963.png)